samples/sample_common/src/sample_utils.cpp

```diff
@@ -779,6 +779,7 @@ mfxStatus CJPEGFrameReader::ReadNextFrame(mfxBitstream *pBS)
 CIVFFrameReader::CIVFFrameReader()
 {
     MSDK_ZERO_MEMORY(m_hdr);
+    MSDK_ZERO_MEMORY(m_hInfo);
```

Comment:
nit: I'd rather added default in-place initialization for ```BitrstreamHeaderInfo``` class

Reply:
Done